src/TTrigger.cpp

```diff
@@ -301,7 +305,7 @@ bool TTrigger::match_perl(char* subject, const QString& toMatch, int regexNumber
     pcre_fullinfo(re.data(), nullptr, PCRE_INFO_NAMECOUNT, &namecount);
 
     if (namecount <= 0) {
-        ;
+        ;// Do something?
     } else {
```

Comment:
💡 Someone should just invert the logic of the test and move the `else` branch into the `then` branch area...!